src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -85,8 +85,21 @@ typedef enum {
 } EnHorseAnimationIndex;
 
 typedef enum {
-    /* 0 */ HORSE_EPONA,
-    /* 1 */ HORSE_HNI
+    /* 0 */ ENHORSE_EPONA,
+    /* 1 */ ENHORSE_HNI
+} EnHorseType;
+
+/**
+ * HorseType values that are passed to Actor_Spawn#param
+ */
+typedef enum {
+    /* 1 */ HORSE_TYPE_1 = 1,
+    /* 2 */ HORSE_TYPE_2,
+    /* 5 */ HORSE_TYPE_5 = 5,
+    /* 6 */ HORSE_TYPE_6,
+    /* 7 */ HORSE_TYPE_7,
+    /* 8 */ HORSE_TYPE_8,
+    /* 9 */ HORSE_TYPE_9
 } HorseType;
```

Comment:
`EnHorse.type` / `EnHorseType` is already a "horse type" so it would be highly confusing to have two horse types

This enum should get a different enough name. Maybe EnHorseParam (since mainly used for params) but I don't know if that would be any good.

Also this enum should get used in z_en_horse.c as well